sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/KafkaBundleWithConfigIT.java

```diff
@@ -531,9 +455,6 @@ public void kafkaConsumerReceivesMessages() {
                     new IgnoreAndProceedErrorHandler<>()))
             .build());
 
-    // empty topic before test
-    KAFKA.getKafkaTestUtils().consumeAllRecordsFromTopic(topic);
```

Comment:
This is a new topic, so we can skip this. Saves us >3 seconds in execution and a potential failure-opportunity.